lib/Mojo/Pg/Results.pm

```diff
@@ -2,30 +2,34 @@ package Mojo::Pg::Results;
 use Mojo::Base -base;
 
 use Mojo::Collection;
-use Mojo::JSON 'from_json';
 use Mojo::Util 'tablify';
 
 has 'sth';
+has expanders => sub { {} };
```

Comment:
Also not ordered alphabetically. And i don't think this attribute needs a default value.